src/components/menu/MenuOption/index.tsx

```diff
@@ -0,0 +1,121 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import { DeleteIcon, EditIcon } from "components/icons";
+import useClickOutside from "hooks/useClickOutside";
+import { HTMLAttributes, useState } from "react";
+import colors from "styles/color";
+import fonts from "styles/font";
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  title: string;
+  options: { name: string; price: number }[];
+}
+
+const Option = ({ title, options, ...rest }: Props) => {
+  const [isSelected, setSelected] = useState<boolean>(false);
```

Comment:
state, set+State 컨벤션으로 작성은 안하시는 편이실까요~? 다른 파일의 상태명에서도 `  const [originalMetadata, setMetadata] = useState<ImageMetadata>();` 이렇게 표현된 게 있어서 여쭤봅니당!
```suggestion
  const [isSelected, setIsSelected] = useState<boolean>(false);
```

Reply:
그리고 상태값이 primitive type일때도 TS에 자동추론을 맡기지 않고 Generic으로 boolean과 같이 타입을 잡아주시는 코드 스타일을 선호하시는지 궁금합니다!

Reply:
```sonarQube```라는 정적분석기에서는

```useState call is not destructured into value + setter pair``` 규칙이 따로 있는 만큼 큰 이유가 없다면  @Seung-wan이 말해주신 ```set+state```방식으로 표현하는게 좋을거 같습니다.

vscode 익스텐션으로 미리 정적분석을 하면서 코딩을 진행 할 수도 있어요. 은근히 유용하더라고요, 코드 스멜이나면 잡아주거나 합니다.

 > https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode

Reply:
> 그리고 상태값이 primitive type일때도 TS에 자동추론을 맡기지 않고 Generic으로 boolean과 같이 타입을 잡아주시는 코드 스타일을 선호하시는지 궁금합니다!

승완님의 꼼꼼한 리뷰에 매번 놀라고 갑니다~

Reply:
@Seung-wan 원래는 set+state로 저도 했었거든요? 그런데 최근에 더 명시적으로 알려줘야해서 만들어진걸 그대로 쓰지 말라는 글을 봤어서, 변경해봣는데 솔직히 전 set+state가 더나은거같아요 ㅋㅋㅋ useState랑 세트라는 느낌도들고

Reply:
@Seung-wan 딱히 타입추론을 덜 선호하냐 이런건 아니고 둘다 상관없는데 전 그냥 명시하는편이에요! 혹시나 하는 실수를 방지하기 위할뿐...